code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -934,13 +934,87 @@
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
 		if (iscultist(H))
-			H.Dizzy(6)
-			H.Jitter(12)
-			if (prob(20))
-				H.Knockdown(1)
-			else if (prob(30))
-				H.confused = 2
-			H.eye_blurry = max(H.eye_blurry, 3)
+			var/unholy = 0
+			var/current_act = max(-1,min(5,veil_thickness))
+			var/datum/role/cultist/cult = H.mind.GetRole(CULTIST)
+			for (var/T in cult.tattoos)
+				var/datum/cult_tattoo/tattoo = cult.tattoos[T]
+				if (tattoo && istype(tattoo, /datum/cult_tattoo/fast))
+					unholy = 1
+			if (cult.holywarning_cooldown <= 0)
+				cult.holywarning_cooldown = 5
+				if (unholy)
```

Comment:
Why are the message to the guy and the actual effect in different switches ; why not do it all in one go